apps/web/src/pages/api/config/[projectId]/index.ts

```diff
@@ -0,0 +1,216 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { prisma } from "server/db/client";
+import NextCors from "nextjs-cors";
+import { z } from "zod";
+import { FlagService } from "server/services/FlagService";
+import { TestService } from "server/services/TestService";
+import { hashApiKey } from "utils/apiKey";
+import {
+  abbyConfigSchema,
+  getDefaultFlagValue,
+  stringifyFlagValue,
+} from "@tryabby/core";
+import {
+  transformClientFlagToDBType,
+  transformDBFlagTypeToclient,
+} from "lib/flags";
+import { AbbyConfigFile } from "@tryabby/core";
+
+const incomingQuerySchema = z.object({
+  projectId: z.string(),
+});
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  await NextCors(req, res, {
+    methods: ["GET", "PUT"],
+    origin: "*",
+    optionsSuccessStatus: 200,
+    allowedHeaders: ["Authorization"],
+  });
+  const querySchemaResult = incomingQuerySchema.safeParse(req.query);
+
+  if (!querySchemaResult.success) {
+    res.status(400).json({ error: "Invalid Query Parameters" });
+    return;
+  }
+
+  const { projectId } = querySchemaResult.data;
+
+  const apiKey = req.headers["authorization"]?.split(" ")[1];
```

Comment:
allowed headers says `Authorization`? is next somehow lower-casing them?

Reply:
yup,  that's whats happening :(